src/apiRedux/dishes.js

```diff
@@ -1,6 +1,8 @@
 import { config } from '../config'
 
-import { getPOSTparams, getPUTparams, getDELETEparams } from 'assets'
+import { getPOSTparams, getPUTparams, getDELETEparams } from './helpers'
+
+const urlBasket = 'basketDishes'
```

Comment:
Tak krótki helper nie ma za bardzo sensu, bo:
- finalnie i tak piszesz tyle samo kodu :D a to jest tylko w tym pliku
- czasem warto mieść dosłownie coś napisane i w razie czego podmienić w kilku miejscach, zamiast robić zmienne pomocnicze
Dałbym po prostu niżej w urlach basketDishes zamiast ${urlBasket}

Reply:
W pliku niżej widzę podobnie zrobiłeś i przez to na pierwszy rzut oka te endpointy robią to samo, dopiero jak się zagłębisz w zmienne, to widzisz że jest inny adres. Minusy przeważają nad plusami w takim wypadku :P

Reply:
Okej, okazało się że miałem literówkę w jednym i długo szukałem przyczyny więc posiadając 3 te same końcówki zdecydowałem się na to rozwiązanie.
Pomijam.